src/modules/translation/languages/en.json

```diff
@@ -72,7 +72,10 @@
     "promo_assetpack_banner": "The Fantasy Asset Pack allows you to conjure all manner of dazzling creations!",
     "promo_body": "Get the Fantasy Asset Pack and conjure all manner of dazzling creations!",
     "promo_subtitle": "Where the magic happens!",
-    "promo_title": "Decentraland"
+    "promo_title": "Decentraland",
+    "top_banner_left":"Sign up for our upcoming",
+    "top_banner_action":"Hackathon!",
```

Comment:
just double checking, I thought we dropped the "hackathon" in favor of "game jam", is this copy ok?

Reply:
yes, it is ok